core/kazoo_caches/src/kz_cache.erl

```diff
@@ -78,11 +70,13 @@
                ,new_node_flush = 'false' :: boolean()
                ,expire_node_flush = 'false' :: boolean()
                ,expire_period = ?EXPIRE_PERIOD :: timeout()
-               ,expire_period_ref :: reference()
+               ,expire_period_ref :: reference() | 'undefined'
                ,props = [] :: kz_term:proplist()
                ,has_monitors = 'false' :: boolean()
                }).
+
 -type state() :: #state{}.
+-export_type([store_options/0, state/0]).
```

Comment:
Why would `#state{}` be exported as a type? It is internal to the gen_server, no?